.github/workflows/build-test-publish.yml

```diff
@@ -13,7 +13,7 @@ on:
     branches: [ "main" ]
 
 env:
-  GO_VERSION: ~1.24
+  GO_VERSION: 'stable'
```

Comment:
We should presumably include some sort of equivalent functionality in the Dockerfile or we'll run into problems in the build stage